### ISSUE TITLE
compile error in `make depend`: cogen.ml#L9

### ISSUE THREAD

**Issue by jeehoonkang** [SoftwareFoundationGroupAtKyotoU/SystemFg#2]
After installing `menhir`, I met the following error message:

```
$ ocamlc --version
4.05.0

$ make depend
ocamldep  -native *.mli *.ml > .depend
File "cogen.ml", line 9, characters 2-3:
Error: Syntax error
make: *** [depend] Error 2
```

**Comment by jeehoonkang** [comment]
It seems `omake` works fine.

**Comment by aigarashi** [comment]
`cogen.ml` is actually written in MetaOCaml and I have been using `ocamldep` adapted to MetaOCaml without noticing.  Fixed in 65dfc1b.  Thanks!
